clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -70,18 +75,37 @@ public void testSerialization() throws Exception {
                 createProduceResponse(),
                 createStopReplicaRequest(),
                 createStopReplicaRequest().getErrorResponse(0, new UnknownServerException()),
-                createStopReplicaResponse());
+                createStopReplicaResponse(),
+                createUpdateMetadataRequest(1),
+                createUpdateMetadataRequest(1).getErrorResponse(0, new UnknownServerException()),
+                createUpdateMetadataResponse(),
+                createLeaderAndIsrRequest(),
+                createLeaderAndIsrRequest().getErrorResponse(0, new UnknownServerException()),
```

Comment:
I checked this and I think 0 is correct for `LeaderAndIsrResponse`. Or am I missing something?